src/lib/openjp2/CMakeLists.txt

```diff
@@ -87,7 +87,9 @@ if(WIN32)
     add_definitions(-DOPJ_STATIC)
   endif()
 endif()
-add_library(${OPENJPEG_LIBRARY_NAME} ${OPENJPEG_SRCS})
+add_library(${OPENJPEG_LIBRARY_NAME} SHARED ${OPENJPEG_SRCS})
+add_library(openjp2_static STATIC ${OPENJPEG_SRCS})
```

Comment:
https://stackoverflow.com/questions/2152077/is-it-possible-to-get-cmake-to-build-both-a-static-and-shared-version-of-the-sam and https://datainfer.wordpress.com/2013/10/24/make-both-static-and-shared-libraries-in-one-build-with-cmake/ give one hint how to do that with set_target_properties(MyLibStatic PROPERTIES OUTPUT_NAME MyLib), but this one must be restricted to non-Windows platform, as Windows apparently need different names.